fix(vikunja): jwt length in string

marcofaggian opened the pull request:
By moving from int to string, we can prevent the issue of an automatic convertion of numbers to exponential notation.
The ladder is not interpreted correctly by the service, that errors out and generates a JWT with "now" expiry.

Commit:
preocess deletes (+1 squashed commits)

Squashed commits:

[f0af9b2aea] squashed previous work on stable train

Commit:
Update charts-test.yaml

Signed-off-by: Kjeld Schouten-Lebbing <kjeld@schouten-lebbing.nl>

Commit:
Update catalog-test.yaml

Signed-off-by: Kjeld Schouten-Lebbing <kjeld@schouten-lebbing.nl>

xstar97 committed:
housekeeping + probes A to F

stavros-k committed:
run precoomit

marcofaggian committed:
fix(vikunja): jwt length in string

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/truecharts/charts?pullRequest=7863) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/truecharts/charts?pullRequest=7863) before we can accept your contribution.<br/>**4** out of **5** committers have signed the CLA.<br/><br/>:white_check_mark: xstar97<br/>:white_check_mark: Ornias1993<br/>:white_check_mark: marcofaggian<br/>:white_check_mark: stavros-k<br/>:x: Marco Faggian<br/><hr/>**Marco Faggian** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/truecharts/charts?pullRequest=7863) it.</sub>

stavros-k commented:
I prefer this handled in the chart itself by helm. Cause it native helm users will also have this problem.

marcofaggian commented:
> I prefer this handled in the chart itself by helm. Cause it native helm users will also have this problem.

Right, I've changed here #[3d29851](https://github.com/truecharts/charts/pull/7863/commits/3d2985152aaf8eb09b8aa9233b996a666a39d8b8) from converting to int, to converting to quotations, should resolve the root issue!

stavros-k commented:
> > I prefer this handled in the chart itself by helm. Cause it native helm users will also have this problem.
> 
> Right, I've changed here #[3d29851](https://github.com/truecharts/charts/pull/7863/commits/3d2985152aaf8eb09b8aa9233b996a666a39d8b8) from converting to int, to converting to quotations, should resolve the root issue!

Check https://github.com/truecharts/charts/blob/39a2f1568fcad35a0bb8efa1c5f2e4a64e07db8b/charts/incubator/radicale/templates/_configmap.tpl#L43

marcofaggian committed:
fix(vikunja): multiply by 1 to avoid scientific notation

stavros-k commented on charts/stable/vikunja/templates/_secrets.tpl:
```diff
@@ -51,8 +51,12 @@ stringData:
-      jwtttl: {{ .Values.vikunja.service.jwtttl | int }}
-      jwtttllong: {{ .Values.vikunja.service.jwtttllong | int }}
+      {{- /*
+        Multiply by 1, so large integers aren't rendered in scientific notation
```
```suggestion
        Multiply by 1, so large integers are not rendered in scientific notation
```
Just because gh highlighting issue

stavros-k commented:
Changed the target to the migration PR, can you please rebase or fix conflict. So we can merge it now
Forgot to mention initially.
Thanks

stavros-k committed:
Stablefixesport (#7866)

**Description**
<!--
Please include a summary of the change and which issue is fixed. Please
also include relevant motivation and context. List any dependencies that
are required for this change.
-->
⚒️ Fixes  # <!--(issue)-->

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing
functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**
<!--
Please describe the tests that you ran to verify your changes. Provide
instructions so we can reproduce. Please also list any relevant details
for your test configuration
-->

**📃 Notes:**
<!-- Please enter any other relevant information here -->

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand
areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is
effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic
versioning

**➕ App addition**

If this PR is an app addition please make sure you have done the
following.

- [ ] 🪞 I have opened a PR on
[truecharts/containers](https://github.com/truecharts/containers) adding
the container to TrueCharts mirror repo.
- [ ] 🖼️ I have added an icon in the Chart's root directory called
`icon.png`

---

_Please don't blindly check all the boxes. Read them and only check
those that apply.
Those checkboxes are there for the reviewer to see what is this all
about and
the status of this PR with a quick glance._

---------

Signed-off-by: Stavros Kois <47820033+stavros-k@users.noreply.github.com>

Crow-Control reviewed (changes requested):
blocked till common refactor on stable is done.

Commit:
bump common and redis

Commit:
rebased

Signed-off-by: Marco Faggian <marco.faggian@leitner.com>

stavros-k commented:
Closing as this changes included in the mentioned commit.
Thanks

stavros-k commented:
@all-contributors please add @marcofaggian for code

truecharts-admin commented:
This PR is locked to prevent necro-posting on closed PRs. Please create a issue or contact staff on discord if you want to further discuss this
